modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java

```diff
@@ -234,25 +406,78 @@ public void testGetObjectEntry() throws Exception {
 
 		Assert.assertEquals(
 			title, MapUtil.getString(objectEntry.getProperties(), "title"));
+	}
 
-		_objectEntryManager.deleteObjectEntry(
-			TestPropsValues.getCompanyId(), _getDTOConverterContext(),
-			objectEntry.getExternalReferenceCode(), _objectDefinition,
-			ObjectDefinitionConstants.SCOPE_COMPANY);
+	private void _assertEquals(
+		List<ObjectEntry> actualObjectEntries,
+		List<ObjectEntry> expectedObjectEntries) {
+
+		Assert.assertEquals(
+			actualObjectEntries.toString(), expectedObjectEntries.size(),
+			actualObjectEntries.size());
+
+		for (int i = 0; i < expectedObjectEntries.size(); i++) {
+			_assertEquals(
+				actualObjectEntries.get(i), expectedObjectEntries.get(i));
+		}
+	}
+
+	private void _assertEquals(
```

Comment:
Maybe we can also reuse this method in a common class (BaseObjectEntryManagerImplTest). What do you think?